libs/dynamixel/dynamixel.ts

```diff
@@ -0,0 +1,214 @@
+namespace network {
+    export class DYNAMIXEL {
```

Comment:
Rename to Dynamixel — classes are capitalized

Reply:
Done on 86f708f.